keps/sig-api-machinery/20190226-network-proxy.md

```diff
@@ -112,19 +112,21 @@ Direct IP routability between cluster and master networks should not be assumed.
 Later version may relax the all node requirement to some.
 - **KAS** Kubernetes API Server, responsible for serving the Kubernetes API to clients.
 - **KMS** Key Management Service, plugins for secrets encryption key management
-- **Connectivity service** A component built into the KAS which provides a golang dialer for outgoing connection requests.
+- **Konnectivity service** A component built into the KAS which provides a golang dialer for outgoing connection requests.
```

Comment:
I thought the things running in the cluster were the konnectivity servers/agents, and the code in apiserver was an egress selector?